### ISSUE TITLE
Bittrex doesn't have LTC-USD 

### ISSUE THREAD

**Issue by terencetsao1987** [ttsao1987/arbitrage#2]
For an exchange that doesn't have a particular ticket, we need to design better logic to skip ONLY for that exchange. Right now it's commented out to skip for all exchanges

**Comment by rhlsthrm** [comment]
Yes, this shouldn't matter, we should make it check if the ticker exists for the exchange, and if it does, then compare to the other exchanges.
